File: Angular App/src/Models/Exam.ts
import { Answer } from "./Answer";
import { Question } from "./Question";

export interface Exam{
  id:number ,
  name:string,
    description:string,
    finalDegree:number;
    imgurl:string,
    duration:number
  questions:Question[]
  havePassed:boolean
 


}

File: API/OnlineExamAPI/DAL/Entities/Exam.cs
using System.ComponentModel.DataAnnotations.Schema;
namespace OnlineExamAPI.DAL.Entities
{
    public class Exam
       {[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string name { get; set; }
        public string description { get; set; } 
        public int FinalDegree { get; set; }
        public IEnumerable<UserExam> userExams { get; set; }
        public IEnumerable<Question> questions { get; set; }
        public string imgurl { get; set; }   
        public int duration { get; set; }

    }
}
